driver/mounter_test.go

```diff
@@ -0,0 +1,137 @@
+package driver
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"gotest.tools/v3/assert"
+)
+
+type testAttachmentValidator struct {
+	readFileFunc     func(name string) ([]byte, error)
+	evalSymlinksFunc func(path string) (string, error)
+}
+
+func (av *testAttachmentValidator) readFile(name string) ([]byte, error) {
+	return av.readFileFunc(name)
+}
+
+func (av *testAttachmentValidator) evalSymlinks(path string) (string, error) {
+	return av.evalSymlinksFunc(path)
+}
+
+func Test_mounter_IsAttached(t *testing.T) {
+	testSource := "test-source"
+	testEvalSymlinkErr := errors.New("eval sym link err")
+	testReadFileErr := errors.New("read file err")
+
+	tests := []struct {
+		name     string
+		av       AttachmentValidator
+		errorMsg string
+		wantErr  bool
```

Comment:
`wantErr` is not really needed since it derives from `errorMsg != ""`.

Reply:
I've removed the wantErr and want properties and now I'm only relying on `errorMsg`.